app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt

```diff
@@ -0,0 +1,191 @@
+import com.bridgewell.bwmobile.ads.inapp.InAppApi
+import com.bridgewell.bwmobile.ads.inapp.listener.BwsAdViewListener
+import com.bridgewell.bwmobile.ads.inapp.model.BwsAdView
+import com.bridgewell.quickstart.android.R
+import com.bridgewell.quickstart.android.activity.ui.adapter.TabbedViewAdapter
+import com.bridgewell.quickstart.android.data.AdType
+import com.bridgewell.quickstart.android.utils.showToast
+import com.google.android.material.tabs.TabLayout
+import com.google.android.material.tabs.TabLayoutMediator
+import koleton.api.hideSkeleton
+import koleton.api.loadSkeleton
+import org.prebid.mobile.api.exceptions.AdException
+import timber.log.Timber
+
+class TabbedViewActivity : AppCompatActivity() {
+
+    companion object {
+        const val CONFIG_ID_BANNER = "dev-bws-banner-ad"
+        const val CONFIG_ID_MOBILE_STICKY_BOTTOM = "dev-bws-mobile-sticky-ad"
+    }
+
+    private val inAppApi = InAppApi()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_tabbed_view)
+
+        val tabLayout = findViewById<TabLayout>(R.id.tabLayout)
+        val viewPager = findViewById<ViewPager2>(R.id.viewPager)
+
+        val adapter = TabbedViewAdapter(this)
+        viewPager.adapter = adapter
+
+        val backButton = findViewById<ImageButton>(R.id.backButton)
+        backButton.setOnClickListener {
+            this.onBackPressedDispatcher.onBackPressed()
+        }
+
+        TabLayoutMediator(tabLayout, viewPager) { tab, position ->
+            when (position) {
+                0 -> {
+                    tab.text = "Home"
+                    tab.setIcon(R.drawable.ic_home) // Set icon for Home tab
+                }
+                1 -> {
+                    tab.text = "News"
+                    tab.setIcon(R.drawable.ic_news) // Set icon for News tab
+                }
+                2 -> {
+                    tab.text = "Files"
+                    tab.setIcon(R.drawable.ic_files) // Set icon for Files tab
+                }
+                3 -> {
+                    tab.text = "User"
+                    tab.setIcon(R.drawable.ic_user) // Set icon for User tab
+                }
+            }
+        }.attach()
+
+        // Set default selected tab
+        viewPager.setCurrentItem(0, false)
+
+        // Get the selected tab index from the intent
+        val selectedTabIndex = intent.getIntExtra("SELECTED_AD_TYPE", 0)
+
+        // Parse the index back to the AdType enum
+        val selectedAdType = AdType.entries[selectedTabIndex]
+
+        for (i in 1..17) {
+            val imageViewId = resources.getIdentifier("imageView$i", "id", packageName)
+            val imageView = findViewById<ImageView>(imageViewId)
+            val skeletonColor = when (i) {
+                1, 2, 5, 6, 11, 10 -> R.color.skeleton_gray_1
+                else -> R.color.skeleton_gray_2
+            }
+            imageView.loadSkeleton {
```

Comment:
![image](https://github.com/user-attachments/assets/e0257aab-33f5-47dc-a0aa-0c866d4766ef)
Does my IDE need any adjustments?

Reply:
Yes, please sycn the project. Gradle files has changed